runtime/JavaScript/src/antlr4/misc/BitSet.d.ts

```diff
@@ -0,0 +1,30 @@
+export declare class BitSet {
```

Comment:
This has nothing to do with ErrorListener, and cannot come with this PR.
Moreover, BitSets are rather internal, so we'd need to understand your use case.

Reply:
I had to add this because, [ErrorListener needs this type](https://github.com/antlr/antlr4/pull/4731/files#diff-b065b124893ec9bbe2eaa123fbeb953e080bb1ecb9cf81b250bd215b0ceaa90fR2)

Reply:
done